### DISCUSSION TITLE
posts/Go/use_zap_in_gin/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#336] [Announcements]
https://www.liwenzhou.com/posts/Go/use_zap_in_gin/ 

 李文周的Blog gin zap 日志 归档 切割 配置管理 go web 框架 收集日志 日志采集

**Comment by kevinniu666** [comment]
请问logger.InitLogger 这个函数的源码在那里呢？或者使用zap结合lumberjack的整个源码

**Comment by jxy90** [comment]
少了部分LogConfig代码。例如：config.Init

**Comment by Q1mi** [comment]
> @kevinniu666
> 请问logger.InitLogger 这个函数的源码在那里呢？或者使用zap结合lumberjack的整个源码

logger.InitLogger代码在上面有的呀...

**Comment by Q1mi** [comment]
> @jxy90
> 少了部分LogConfig代码。例如：config.Init

感谢反馈，已添加完整示例代码。

**Comment by weijunh** [comment]
main函数的
```
if err := config.Init(os.Args[1]); err != nil {
		panic(err)
	}
```
就报错panic: runtime error: index out of range [1] with length 1
请问这个怎么用??

**Comment by asdlin** [comment]
> @weijunh
> main函数的
> ```
> if err := config.Init(os.Args[1]); err != nil {
> 		panic(err)
> 	}
> ```
> 就报错panic: runtime error: index out of range [1] with length 1
> 请问这个怎么用??

编译后 执行exe文件 后面带上config.json

**Comment by cmh1779** [comment]
建议作者有空还是补一些注释，小白看得实在头疼，云里雾里的，视频也没有详细介绍。
